cloudbuild.yaml

```diff
@@ -3,17 +3,62 @@
 
 steps:
 # Perform maven build, omitting local docker operations
-- name: 'maven:3.3.9-jdk-8'
-  args: ['mvn', '-B', '-P-local-docker-build', '-P-test.local', '-Ddocker.tag.long=$_DOCKER_TAG', 'clean', 'install']
+- name: 'gcr.io/cloud-builders/java/mvn:3.5.0-jdk-8'
+  args:
+    - '--batch-mode'
+    - '-P-local-docker-build'
+    - '-P-test.local'
+    - '-Ddocker.tag.long=${_DOCKER_TAG}'
```

Comment:
It might be judicious to override properties such as the runtime name and version in the maven configuration.

Otherwise we could publish a version that differ from the version in the tag.